openshift/datagrid-service/user-config/README.md

```diff
@@ -0,0 +1,208 @@
+  --restart=OnFailure \
+  --env SVC_DNS_NAME=datagrid-service-user-config-hotrod \
+  --env JAVA_OPTIONS=-ea
+```
+  The quickstart application connects to the Data Grid service and invokes a `PUT` operation that stores a `key='key-hotrod'/value='user-config'` pair and performs an assertion to verify the entry.
+
+2. Verify the `PUT` operation completed successfully.
+```
+$ oc logs quickstart --tail=50
+--- Connect to datagrid-service-user-config-hotrod ---
+...
+--- Store key='key-hotrod'/value='user-config' pair ---
+...
+--- Retrieve key='key-hotrod' ---
+--- Value is 'user-config' ---
+```
+
+3. Delete resources for the quickstart application.
+    ```bash
+    $ oc delete all --selector=run=quickstart || true
+    $ oc delete imagestream quickstart || true
+    $ oc delete buildconfig quickstart || true
+    ```
+
+Verifying Access with HTTP
+--------------------------
+1. Invoke a `POST` operation to store data via the REST API over HTTP.
+```bash
+$ oc exec \
+  -it datagrid-service-user-config-0 \
+  -- curl -v -X POST \
+  -H 'Content-type: text/plain' \
+  -d 'user-config' \
+  datagrid-service-user-config-http:8080/rest/default/key-rest
+```
+
+  If the `POST` operation is successful, you should get output similar to the following:
+
+  ```
+  * About to connect() to datagrid-service-user-config-http port 8080 (#0)
+  *   Trying 172.30.180.240...
+  * Connected to datagrid-service-user-config-http (172.30.180.240) port 8080 (#0)
+  > POST /rest/default/key-rest HTTP/1.1
+  > User-Agent: curl/7.29.0
+  > Host: datagrid-service-user-config-http:8080
+  > Accept: */*
+  > Content-type: text/plain
+  > Content-Length: 11
+  >
+  * upload completely sent off: 11 out of 11 bytes
+  < HTTP/1.1 200 OK
+  < connection: keep-alive
+  < etag: -1432381597
+  < content-length: 0
+  <
+  * Connection #0 to host datagrid-service-user-config-http left intact
+  ```
+
+2. Invoke a `GET` operation to retrieve data via the REST API over HTTP.
+```bash
+$ oc exec \
+  -it datagrid-service-user-config-0 \
+  -- curl -v \
+  datagrid-service-user-config-http:8080/rest/default/key-rest
+```
+
+  If the `GET` operation is successful, you should get output similar to the following:
+
+  ```
+  * About to connect() to datagrid-service-user-config-http port 8080 (#0)
+  *   Trying 172.30.180.240...
+  * Connected to datagrid-service-user-config-http (172.30.180.240) port 8080 (#0)
+  > GET /rest/default/key-rest HTTP/1.1
+  > User-Agent: curl/7.29.0
+  > Host: datagrid-service-user-config-http:8080
+  > Accept: */*
+  >
+  < HTTP/1.1 200 OK
+  < connection: keep-alive
+  < etag: -1432381597
+  < last-modified: Thu, 1 Jan 1970 00:00:00 GMT
+  < content-type: application/octet-stream
+  < content-length: 11
+  <
+  * Connection #0 to host datagrid-service-user-config-http left intact
+  ```
+
+    You've successfully completed this tutorial!
+
+    Delete the project to remove all resources, for example:
+    ```bash
+    $ oc delete project my_project
```

Comment:
You probably want to extend this section like in other READMEs? e.g. https://github.com/jboss-developer/jboss-jdg-quickstarts/blob/openshift-tutorials/openshift/cache-service/create-cache/README.md